.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "lib/enyo-webos"]
 	path = lib/enyo-webos
 	url = https://github.com/enyojs/enyo-webos
+[submodule "lib/DOMPurify"]
```

Comment:
Is there any specific reason to use git submodule instead of npm package?

Reply:
That's a good question. Need to test it out - no idea how well `enyo-dev` (our current frontend packager) supports pulling in npm packages.

Reply:
@Informatic on my local machine, npm package works. (I thought you were working on case insensitive file system so I added npm package, and changed it to lowercase, and looks like it's working)

Reply:
Nope, I am all Linux-based so that shouldn't be a problem, I just did what I assumed would work the most flawlessly (I just have no idea how `enyo-dev` internals work, and just happened to know that pulling in dependencies from `lib` works :D)

If that works, we could also just migrate existing enyo dependencies submodules into npm dependencies. I might test that out later today.

Reply:
Fixed, pulled in via npm now